grouper/ctl/sync_db.py

```diff
@@ -63,6 +67,27 @@ def sync_db_command(args):
 
         session.commit()
 
+    auditors_group_name = get_auditors_group_name(settings)
+    auditors_group = Group.get(session, name=auditors_group_name)
+    if not auditors_group:
+        auditors_group = Group(
+                groupname=auditors_group_name,
+                description="Group for auditors, who can be owners of audited groups.",
+                canjoin="canjoin",
+        )
+
+        try:
+            auditors_group.add(session)
+            session.flush()
+        except IntegrityError:
+            session.rollback()
+            raise Exception('Failed to create group: {}'.format(auditors_group_name))
+
+        permission = Permission.get(session, PERMISSION_AUDITOR)
+        assert permission, "Permission should have been created earlier!"
```

Comment:
Nit: move this check before the group creation just in case it fails I guess

Reply:
nah it's fine. following the pattern above with the admin group. also this is to init a db for dev, so whatevs :P